versioned_docs/version-1.8/operator-manual/telemetry/opentelemetry/01-quickstart.md

```diff
@@ -130,3 +130,9 @@ helm install --wait \
 
 We can now move to the next chapters to enable application metrics (via Prometheus
 integration) and application tracing (via Jaeger integration).
+
+:::note
+The OpenTelemetry operator must be ready, to create the needed sidecars
```

Comment:
You should remove the comma after ready.

Reply:
Thanks, done in https://github.com/kubewarden/docs/pull/288/commits/f46bac671fa043f8bb9aa06f41ff874a69a3c282/